change/@ni-nimble-components-04071165-e637-4716-ac5d-3c4db6e000cc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Reduce font loading jitter",
```

Comment:
Perhaps something a bit more descriptive for the changelog? E.g. Created `@font-face` definitions for local fonts so they closely match the design system fonts. This helps minimize font resizing jitter on initial page load.

Reply:
Done